src/functions/template-tags/html.php

```diff
@@ -178,31 +179,36 @@ function tribe_format_field_dependency( $deps ) {
 
 		// Handle the ID component.
 		if ( 'id' === $attr ) {
-			$dependency .= " data-depends=\"{$value}\"";
+			$dependency .= " data-depends=\"#{$value}\"";
```

Comment:
Tests need to be updated as this is breaking.

Reply:
This change makes sense in logic, but I'm not sure all of our code would correctly use it and it could result, in some instances, in an attribute value like `##some-id`.
Maybe detect the starting `#` here and add it if required?

Reply:
I'm a bit worried if we have any code currently using this (and other cases of where we're modifying existing code). If so, this could break the current implementation of those instances.

Reply:
Changed

Reply:
@paulmskim please, resolve this conversation when you're satisfied.